### DISCUSSION TITLE
How can I implement lazy loading for a datatable?

### DISCUSSION THREAD

**Discussion by 600888** [flet-dev/flet#2117] [Q&A] [answered]
### Question

How can I implement lazy loading for a table? I have a table with a large amount of data. Every time I start the app, it takes a long time to load, and it's very laggy when switching and scrolling.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by vindevoy** [comment, marked as answer]
Maybe by using some kind of paging tactic ?  Load 20 records and the rest you get handling on_click events in buttons ?  Just an idea.

**Reply by 600888** [reply]
Thank you for your response, I think this is a good method.
